2018-westphal-antpc.tex

```diff
@@ -120,48 +115,48 @@ \subsection{Electroreduction}
+A LiCl-KCl eutectic is used as an electrolyte above 500$^{\circ}$C \cite{flowsheet_1998,lee_korean_2011}. 
+Uranium dissolves at the anode to recombine at the cathode as metallic uranium. 
+Waste transuranics (TRUs) and lanthanides are in a soluble chloride form  while fission products and cladding remain in the anode
+basket. Finally, actinides and fission products are removed from the cladding electrochemically \cite{lee_korean_2011}.
 
-Lee et al. \cite{lee_advanced_2008} show that the addition of vacuum pressure to the system improves removal efficiency. 
+Lee et al. \cite{lee_advanced_2008} show decreasing system pressure improves removal efficiency. 
 Temperature, however, exhibits the opposite effect: as temperature decreases so does salt removal. This comes into effect 
-particularly depending on material choice of instrumentation and containment \cite{lee_advanced_2008}. The most limiting 
-material choice is iron because a eutectic forms between Fe and U at 725$^{\circ}$C \cite{chapman_revision_1984}. 
-In facilities where iron equipment is present temperatures are limited to 700$^{\circ}$C, and efficiency is 
-significantly hindered. Cathode arrangement and anode rotation speed also affect the collection of uranium 
-dendrites \cite{lee_advanced_2008}. The addition of a central stirrer allows mixing of uranium dendrites stuck on 
-the bottom of the vessel, improving separation efficiency and increasing throughput. 
+particularly depending on material choice of instrumentation and containment \cite{lee_advanced_2008}. 
+Iron, for example, limits operating temperature because a eutectic forms at 725$^{\circ}$C \cite{chapman_revision_1984}.
+In facilities where iron equipment is present, temperatures are limited to 700$^{\circ}$C, and efficiency is significantly hindered. 
+Cathode arrangement and anode rotation speed also affect the collection of uranium 
+dendrites \cite{lee_advanced_2008}. A central stirrer mixes uranium dendrites stuck on 
+the vessel, improving separation efficiency and increasing throughput. 
 
 \begin{figure}[ht]
 	\centering
-	\includegraphics[width=0.8\linewidth]{refining}
+	\includegraphics[width=1\linewidth]{refining}
 	\caption{A material balance over the electrorefining sub-process, including observables.}
 	\label{fig:refining}
 \end{figure}
-
-The electrorefining process also produces the fission product waste stream which requires monitoring. The following products are produced 
-and tracked in this step: Tc, Ag, Pd, Rh, Ru, Mo and Zr \cite{flowsheet_1998}. Uranium and TRU product streams separated at this stage are sent to Fuel Fabrication, while the remaining salt is reformed as an oxidant and recirculated.
-Separation efficiencies are taken after recirculation, and treated as a once-through cycle. The Cyclus time step
+ 
+The electrorefining process also produces the fission product waste stream which requires monitoring. 
```

Comment:
Is this the only fission product waste stream? If it's not, then "the fission product waste stream" should be "a fission product waste stream"